app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -151,6 +157,107 @@ const processAuthorizationOptions = (item: PostmanAuth.Item | undefined, parentC
   return auth;
 };
 
+const getContentTypeForRawBody = (bodyType: string) => {
+  switch (bodyType) {
+    case "json":
+      return RequestContentType.JSON;
+    case "text":
+      return RequestContentType.RAW;
+    default:
+      return RequestContentType.RAW;
+  }
+};
```

Comment:
isn't this meant to give the content-type for each preview mode that [postman gives](https://github.com/requestly/requestly/pull/3015#discussion_r2097751425)?

since [you did](https://github.com/requestly/requestly/pull/3015/files#diff-3ed7caa53e23f71cd53c5431d592bff2ee456fec0b022083462e59f3e1b5bbd6R26-R27) add all the possible values of the enum now, why aren't we handling all them here?

Reply:
I understand that you [mentioned](https://github.com/requestly/requestly/pull/3015#discussion_r2097710818) there is a ticket for this
> Adding support for other languages like HTML, XML, JavaScript in request body editor. (Currently Text is default for all unsupported raw types)

but it seems trivial to set the correct content type here, so why not do it?